aries-site/src/layouts/content/Example/Example.js

```diff
@@ -174,10 +174,16 @@ export const Example = ({
   // when Layer is open, we remove the inline Example to avoid
   // repeat id tags that may impede interactivity of inputs
   let content = !fullscreen && (
-    <ExampleContainer as="section" {...containerProps}>
+    <ExampleContainer
+      as="section"
+      {...containerProps}
+      background={
+        !plain ? { color: 'background-contrast', opacity: '0.02' } : undefined
```

Comment:
Where does the opacity come from? What does it enable?

Add code comment?

Reply:
I will add a code comment for whatever we settle on. This came from 'background-contrast' being too "dark" sitting on top of background-back app.

Wanting something that more accurately conveys the "background-back". That said, Mike made a good comment that in dark mode the effect is too subtle.

Wondering if we should take an incremental step in the site-refresh direction where the docs pages are "background" and then the example container can use "background-back"/"background-contrast" to simulate

Reply:
Still thinking on this one.

Reply:
I've made a design proposal for the dark mode treatment in my latest commit and added code comments. LMK your thoughts.